Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -113,7 +113,7 @@ struct SymbolGraphLoader {
         // This strategy benchmarks better when we have multiple
         // "larger" symbol graphs.
         #if os(macOS) || os(iOS)
-        if bundle.symbolGraphURLs.filter({ !$0.path.contains("@") }).count > 1 {
+        if bundle.symbolGraphURLs.filter({ !($0.pathComponents.last ?? "").contains("@") }).count > 1 {
```

Comment:
This seems kind of fragile to me. How confident are we in the stability of the symbol graph file names that are passed to DocC?

Maybe we should file an issue to look into removing the need for this? Based on the comment it seems like we're interested in only concurrently decoding when we have multiple large symbol graphs so maybe we should instead be looking at the size of each graph.

Reply:
There's been no reason to change the symbol graph naming format, and i can only assume that other symbol graph users have come to depend on it by now as well. It's probably worth adding your suggested change, but i'm also looking to get this change landed in release/5.9, so that's probably best as a separate issue/PR.